src/plugins/embeddable/public/lib/panel/_embeddable_panel.scss

```diff
@@ -120,10 +120,14 @@
 // EDITING MODE
 
 .embPanel--editing {
-  border-style: dashed;
-  border-color: $euiColorMediumShade;
+  border-style: dashed !important;
+  border-color: $euiColorMediumShade !important;
   transition: all $euiAnimSpeedFast $euiAnimSlightResistance;
 
+  @include kbnThemeStyle('v8') {
```

Comment:
Can or should this 1px border be applied to both themes?

Reply:
@ryankeairns We could apply this to both themes thinking of the benefit you mentioned. This is only needed in v8 as v7 sets a border width for EuiPanel but doesn't affect v7. Should this become a rule?  ->` when the change is needed for v8 only, set it for both themes provided it doesn't break v7`.

Reply:
That is what I'm thinking yeah. It's not crucial, but it saves us tracking this down and going back through a PR process later. Thanks!